server/src/controllers/ScheduleController.js

```diff
@@ -0,0 +1,88 @@
+const ScheduleServices = require('../services/ScheduleServices');
+
+const ScheduleController = {
+  /**
+   * Get an array of schedule objects for a frosh group
+   * @param {Object} req
+   * @param {Object} res
+   * @param {Function} next
+   * @return {Object[]}
+   */
+  async getGroupSchedule(req, res, next) {
+    const { froshGroup } = req.body;
```

Comment:
`req.body` should not be used in GET requests. In general, bodies should not be sent in a GET request, so trying to access them is an anti-pattern (if technically possible). What you can do is use the `req.user` object which the passportjs middleware is adding, which will give you access to the full mongoose document for the current user